dataset/dictionary/characters.json5

```
@@ -711,80 +732,96 @@
     id: "hutao",
     en: "Hu Tao",
     ja: "胡桃",
+    zhCN: "胡桃",
     pronunciationJa: "フータオ",
     tags: [ "liyue", "character-main" ],
   },
   {
     en: "Yanfei",
     ja: "煙緋",
+    zhCN: "烟绯",
     pronunciationJa: "えんひ",
+    // Pronunciataion of Chinese spelled in Japanese can be 「いぇんふぇい」, IMO. -- Bill Haku
     tags: [ "liyue", "character-main" ],
   },
   // ▼▼ TODO Become a main character in the upcoming version? ▼▼
   {
     en: "Baizhu",
     ja: "白朮",
+    zhCN: "白术",
     pronunciationJa: "びゃくじゅつ",
     tags: [ "liyue", "character-sub" ],
     notes: "英語の発音は「バイジュ」",
   },
   {
     en: "Dr. Baizhu",
     ja: "白先生",
+    zhCN: "白先生",
     tags: [ "liyue", "how-to-call" ],
     notes: "七七などからの白朮の呼び名",
   },
   {
     en: "Yaoyao",
     ja: "ヨォーヨ",
+    zhCN: "瑶瑶",
     tags: [ "liyue", "character-sub" ],
   },
   {
     en: "Xingyan",
     ja: "星燕",
+    zhCN: "星燕",
     tags: [ "liyue", "character-sub" ],
   },
   // ▲▲ TODO Become a main character in the upcoming version? ▲▲
   {
     en: "Yun Jin",
     ja: "雲菫",
+    zhCN: "云堇",
+    // Pronunciataion of Chinese spelled in Japanese can be 「ゆんじん」, IMO. -- Bill Haku
     pronunciationJa: "うんきん",
     tags: [ "liyue", "character-main" ],
   },
   {
     en: "Ms. Yun",
     ja: "雲先生 / 雲座頭",
+    zhCN: "云先生",
     pronunciationJa: "うんせんせい / うんざとう",
     tags: [ "liyue", "how-to-call" ],
     notes: "雲菫への敬称",
   },
   {
     en: "Shenhe",
     ja: "申鶴",
+    zhCN: "申鹤",
     pronunciationJa: "しんかく",
+    // Pronunciataion of Chinese spelled in Japanese can be 「しぇんへぇ」, IMO. -- Bill Haku
```

OK, let's add it as a note!

```suggestion
    notes: "英語及び中国語の読みは「シェンヘェ」",
```